addon/mixins/route.js

```diff
@@ -135,11 +143,21 @@ export default Ember.Mixin.create({
     this.set('_perPage', perPage);
     this.set('_modelPath', modelPath);
     this.set('_extraParams', options);
+    if (typeof boundParams === 'object') {
+      this.set('_boundParams', boundParams);
+    }
 
     var requestPayloadBase = {};
     requestPayloadBase[this.get('perPageParam')] = perPage;
     requestPayloadBase[this.get('pageParam')] = startingPage;
 
+    if (Ember.keys(this.get('_boundParams')).length > 0) {
+      var _this = this;
+      Ember.keys(boundParams).forEach(function(key) {
+        options[key] = _this.get(boundParams[key]);
+      });
+    }
```

Comment:
Can we a) use an ES6 fat arrow function in the `forEach` so we don't have to bind `_this`, and b) pull this out into a private function so we don't have to have it in both `infinityModel` & `infinityLoad`